integrationTests/vm/txDataBuilder.go

```diff
@@ -0,0 +1,127 @@
+package vm
+
+import (
+	"encoding/hex"
+	"math/big"
+
+	"github.com/ElrondNetwork/elrond-go/core"
+)
+
+// TxDataBuilder constructs a string to be used for transaction arguments
+type TxDataBuilder struct {
```

Comment:
we do not need this to be exported, nor its fields as we have a constructor + accessors.

Reply:
Made everything private.